push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -43,6 +43,10 @@
 @SenderType(iOSVariant.class)
 public class APNsPushNotificationSender implements PushNotificationSender {
 
+    private static final String AEROGEAR_APNS_PUSH_HOST = "aerogear.apns.push.host";
+    private static final String AEROGEAR_APNS_PUSH_PORT = "aerogear.apns.push.port";
+    private static final String AEROGEAR_APNS_FEEDBACK_HOST = "aerogear.apns.feedback.host";
+    private static final String AEROGEAR_APNS_FEEDBACK_PORT = "aerogear.apns.feedback.port";
```

Comment:
let's name them different, like `CUSTOM_AEROGEAR....` to be CLEAR that this should be only provided under some criterias (e.g. QE tests).